experimental/generation/generator/packages/library/test/forms/unittest_keyPhrase.schema

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "KeyphraseProperty": {
+            "type": "string",
+            "$entities": [
+                "keyPhrase"
+            ]
```

Comment:
This should be promoted to the top--you do not need an object for a single property.